src/query/db_accessor.hpp

```diff
@@ -597,6 +597,13 @@ class DbAccessor final {
     return accessor_->ApproximateVertexCount(label, property, lower, upper);
   }
 
+  std::vector<storage::LabelId> ListAllPossiblyPresentVertexLabels() const {
```

Comment:
Because it might be possible that the user adds some vertex with a specific label to the database, then deletes it. With these commands we want to keep track of what kind of data `could` be present in the database, hence we do not only check for labels/edge-types that are currently exists in the database.